_reports/2021-05-88mph.md

```diff
@@ -0,0 +1,285 @@
+---
+sponsor: "88mph"
+slug: "2021-05-88mph"
+date: "2021-06-14"
+title: "88mph"
+findings: "https://github.com/code-423n4/2021-05-88mph-findings/issues"
+contest: 7
+---
+
+# Overview
+
+## About C4
+
+Code 432n4 (C4) is an open organization that consists of security researchers, auditors, developers, and individuals with domain expertise in the area of smart contracts.
+
+A C4 code contest is an event in which community participants, referred to as Wardens, review, audit, or analyze smart contract logic in exchange for a bounty provided by sponsoring projects.
+
+During the code contest outlined in this document, C4 conducted an analysis of 88mph’s smart contract system written in Solidity. The code contest took place between May 13 and May 19, 2021.
+
+## Wardens
+
+5 Wardens contributed reports to the 88mph code contest:
+
+- [cmichel](https://twitter.com/cmichelio)
+- [shw](https://twitter.com/shw9453)
+- [gpersoon](https://twitter.com/gpersoon)
+- [Thunder](https://twitter.com/SolidityDev)
```

Comment:
Is this the handle he's registered with for the leaderboard? Has it changed? It was pauliax or paulius

Reply:
Yeah there was some confusion about it last time, @itsmetechjay confirmed this person is Thunder in discord.